client/src/features/ProjectPageV2/ProjectPageContent/ProjectInformation/ProjectInformation.tsx

```diff
@@ -16,41 +16,90 @@
  * limitations under the License.
  */
 
+import { skipToken } from "@reduxjs/toolkit/query";
 import cx from "classnames";
 import { useMemo } from "react";
 import {
   Bookmarks,
   Clock,
+  Diagram3Fill,
   Eye,
   InfoCircle,
   JournalAlbum,
   People,
 } from "react-bootstrap-icons";
 import { Link, generatePath } from "react-router-dom-v5-compat";
 import { Badge, Card, CardBody, CardHeader } from "reactstrap";
+
+import { Loader } from "../../../../components/Loader";
 import { TimeCaption } from "../../../../components/TimeCaption";
-import {
-  EditButtonLink,
-  UnderlineArrowLink,
-} from "../../../../components/buttons/Button";
+import { UnderlineArrowLink } from "../../../../components/buttons/Button";
 import { ABSOLUTE_ROUTES } from "../../../../routing/routes.constants";
 import projectPreviewImg from "../../../../styles/assets/projectImagePreview.svg";
-import PermissionsGuard from "../../../permissionsV2/PermissionsGuard";
 import type {
   ProjectMemberListResponse,
   ProjectMemberResponse,
 } from "../../../projectsV2/api/projectV2.api";
 import {
   useGetNamespacesByNamespaceSlugQuery,
+  useGetProjectsByProjectIdQuery,
   useGetProjectsByProjectIdMembersQuery,
 } from "../../../projectsV2/api/projectV2.enhanced-api";
+import type { Project } from "../../../projectsV2/api/projectV2.api";
 import { useProject } from "../../ProjectPageContainer/ProjectPageContainer";
 import { getMemberNameToDisplay, toSortedMembers } from "../../utils/roleUtils";
 import useProjectPermissions from "../../utils/useProjectPermissions.hook";
+
+import ProjectInformationButton from "./ProjectInformationButton";
 import styles from "./ProjectInformation.module.scss";
 
 const MAX_MEMBERS_DISPLAYED = 5;
 
+function ProjectCopyTemplateInformationBox({ project }: { project: Project }) {
+  const { data: templateProject } = useGetProjectsByProjectIdQuery(
+    project.template_id
+      ? {
+          projectId: project.template_id,
+        }
+      : skipToken
+  );
+  const { data: templateProjectNamespace } =
+    useGetNamespacesByNamespaceSlugQuery(
+      templateProject
+        ? {
+            namespaceSlug: templateProject?.namespace ?? "",
```

Comment:
? This should be fully defined if `templateProject` is truthy:

```suggestion
            namespaceSlug: templateProject.namespace,
```